editor/editor_node.cpp

```diff
@@ -341,6 +341,19 @@ void EditorNode::_update_title() {
 	}
 }
 
+void EditorNode::input(const Ref<InputEvent> &p_event) {
+	Ref<InputEventKey> k = p_event;
+	if (k.is_null()) {
+		return;
+	}
+
+	// Prevent all shortcuts while the progress dialog is displayed
+	// to simulate a exclusive popup.
+	if (progress_dialog->is_visible()) {
```

Comment:
It would be more efficient to call `set_process_input()` when dialog visibility is toggled.

Reply:
It's already what I do, this check was in fact only a precaution in case of something else calls `EditorNode::get_singleton()->set_process_input`. I see that the code was not clear enough. I adjusted the comments and removed the `progress_dialog->is_visible()` condition in `EditorNode::input`.